docs/user/rstudio/ide/guide/tools/jobs-in-action.qmd

```diff
@@ -1,49 +1,199 @@
-# Jobs in action
+---
+title: "Jobs in action"
+---
 
-Copied from: <https://solutions.rstudio.com/r/jobs/>
+As mentioned in [RStudio Jobs](jobs.html), RStudio has the ability to send long running R scripts to [local and remote background jobs](https://www.rstudio.com/blog/rstudio-1-2-jobs/). This functionality can dramatically improve the productivity of data scientists and analysts using R since they can continue working in RStudio while jobs are running in the background.
 
-[RStudio 1.2](https://www.rstudio.com/products/rstudio/) introduced the ability to send long running R scripts to [local and remote background jobs](https://www.rstudio.com/blog/rstudio-1-2-jobs/). This functionality can dramatically improve the productivity of data scientists and analysts using R since they can continue working in RStudio while jobs are running in the background.
+A few examples of integrating RStudio Jobs with specific development workflows in shiny, plumber, machine learning, or any long-running process are documented below.
 
-Local background jobs are supported by all versions of the RStudio, server and desktop. Remote background jobs are a feature of [RStudio Workbench](https://www.rstudio.com/products/workbench/) and are orchestrated by the [Launcher](https://docs.rstudio.com/job-launcher/), which also supports running interactive sessions on remote resource managers like [Kubernetes](https://kubernetes.io/). More information about setting up and adminstering the Launcher can be found [here](https://solutions.rstudio.com/launcher/overview).
+## Shiny app
```

Comment:
So as of the in progress release, we will also be able to run Python Shiny apps as jobs -- perhaps we want to update this section or at least explicitly refer to Shiny for R here?